src/crawl.js

```diff
@@ -184,11 +189,43 @@ async function screenshot(page, filename) {
   if (!disableMobileScreenshots) {
     await page.emulate(puppeteeriPhone11);
 
+    // This has to be called after the emulate to iPhone to work
+    await scrubInstallAndVersion(page);
+
     await page.screenshot({
       path: `${filename}_mobile.png`,
       fullPage: true
     });
   }
 }
 
+/**
+ * The version, install, and usage can be different between the baseline and current
+ * so this will normalize to prevent false positives.
+ */
+async function scrubInstallAndVersion(page) {
+  if (scrubHeader) {
+    await page.evaluate(() => {
+      // removing the install package name
+      let install = document.body.querySelector('td code');
+      if (install && install.innerHTML && install.innerHTML.includes && install.innerHTML.includes('yarn add') && install.parentElement.parentElement.firstChild.innerHTML.includes('install')) {
```

Comment:
It would be since to switch from running this with node 12 to node 14 to use the question mark option chaining.